ops/docker_build.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
```

Comment:
Looking at the code, it's fair to say that we don't need to interact with this script outside of CI, right?

Reply:
It's still useful as a shortcut for `docker_build.py`.